ssl/ssl_lib.c

```diff
@@ -333,7 +333,7 @@ SSL *SSL_new(SSL_CTX *ctx)
 # ifndef OPENSSL_NO_EC
     if (ctx->tlsext_ecpointformatlist) {
         s->tlsext_ecpointformatlist =
-            BUF_memdup(ctx->tlsext_ecpointformatlist,
+            OPENSSL_memdup(ctx->tlsext_ecpointformatlist,
```

Comment:
The indentation on the continuation line ought to be adjusted here and for the next line (maybe elsewhere?)